tests/api_workflow/test_api_workflow_agent.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+
+from lightly.active_learning.agents.agent import ActiveLearningAgent
+from lightly.active_learning.config.sampler_config import SamplerConfig
+from lightly.active_learning.scorers.classification import ScorerClassification
+from lightly.openapi_generated.swagger_client import SamplingMethod
+from tests.api_workflow.mocked_api_workflow_client import MockedApiWorkflowSetup
+
+
+class TestApiWorkflowAgent(MockedApiWorkflowSetup):
```

Comment:
So is this a test  for the api workflow or the active learning agent? Filename confuses me

Reply:
I renamed it and put it into the active learning folder, but it still uses the mocks of the mocked_api_workflow_client